### DISCUSSION TITLE
Can no longer access QuickBooks Online AU sandbox Companies via Codat since Friday

### DISCUSSION THREAD

**Discussion by russell-clarke** [codatio/codat-docs#887] [Q&A]
Hi all, thanks in advance for reading this. 

We have several existing QBO sandbox companies set up for testing purposes - they are AU region, however since Friday we have been unable to link to them:
![image](https://github.com/codatio/codat-docs/assets/151475968/ba02be53-fae3-40eb-8ab4-cbc7553c670b)

We have created some new US region sandbox companies, and can see them and link to them. When we also created a new AU region sandbox company, it didn't appear in the company selection list:
![image](https://github.com/codatio/codat-docs/assets/151475968/92b4b8b6-1e8f-4f40-a748-74cbd84c6d89)

Is there some type of filtering that Codat is doing all of a sudden based on the region of the QBO company? I have checked through the configs and can't see anything immediately obvious. I am hoping I have missed something!

Note that I am aware that these are screenshots of Intuit pages, but it would appear that what Codat is requesting during authentication/authorisation is where the issue exists. We can see the AU sandbox companies when we log into our QBO account, so the problem doesn't appear to stem from there.

Any assistance would be appreciated :)

**Comment by dcoplowe** [comment]
Hi @russell-clarke,

I'd suggest reaching out to our [support team](https://codat.zendesk.com/hc/en-gb/requests/new) and sharing the companyIds of existing QBO sandbox companies you're unable to link. This will help us assist with triaging your issue.

Many thanks,

David
